zwave_js_server/model/node.py

```diff
@@ -72,10 +75,18 @@ def __init__(self, client: "Client", data: NodeDataType) -> None:
         self.values: Dict[str, Union[Value, ConfigurationValue]] = {}
         for val in data["values"]:
             value_id = _get_value_id_from_dict(self, val)
-            if val["commandClass"] == CommandClass.CONFIGURATION:
-                self.values[value_id] = ConfigurationValue(self, val)
-            else:
-                self.values[value_id] = Value(self, val)
+            try:
+                if val["commandClass"] == CommandClass.CONFIGURATION:
+                    self.values[value_id] = ConfigurationValue(self, val)
+                else:
+                    self.values[value_id] = Value(self, val)
+            except UnparseableValue:
+                # If we can't parse the value, don't store it but log it for later
+                _LOGGER.info(
```

Comment:
Generally it's nice to avoid side effects in init methods. Maybe we should introduce a classmethod to create the node that does the parsing and change the signature of the Node class to accept the values dict too?

Reply:
But to avoid side effects in any of the models we'd have to change all the models up to the driver. 🙈

Reply:
I'm not following this suggestion. Perhaps it would be easier to remove the logging for now?

Reply:
I don't like that big change now. Not sure about this.

Reply:
Yeah that's a much bigger change. I think for now we can just skip the value and we can plan to address that in the future

Reply:
Ok 👍